CONTRIBUTING.rst

```diff
@@ -24,6 +24,15 @@ message guidelines described below:
 
 Please provide your patches as GitHub pull requests. Thanks!
 
+Rerunning failed CI workflows
+-------------------------
```

Comment:
Maybe we can add a sentence here before the list, currently the rendering is a bit odd: 
https://github.com/python-gitlab/python-gitlab/blob/6d98c7588447a7423d7506c6deafb665c642b955/CONTRIBUTING.rst#rerunning-failed-ci-workflows

Reply:
Apparently there is a cool vscode plugin which helped me realize what was wrong. Thanks for pushing towards it.